pkgs/development/python-modules/maestral/default.nix

```diff
@@ -44,8 +45,26 @@ buildPythonPackage rec {
     "--prefix" "PYTHONPATH" ":" "$out/lib/${python.libPrefix}/site-packages"
   ];
 
-  # no tests
-  doCheck = false;
+  checkInputs = [
+    pytestCheckHook
+    pytest-benchmark
```

Comment:
We don't want to benchmark. Can we disable this feature in setup.py/etup,cfg?

Reply:
I've disabled the single `"test_performance"` benchmark, is that good enough? I don't know how to do it from setup.cfg, would it need to be patched?

Reply:
> I've disabled the single "test_performance" benchmark, is that good enough?

That should be fine.

> don't know how to do it from setup.cfg, would it need to be patched?

Yeah, we would most likely do it in postPatch with substituteInPlace.